assets/prototype/domain/eventEditor/containers/useConfigProvider.ts

```diff
@@ -1,30 +1,25 @@
 import { useContext } from 'react';
-import { pick } from 'ramda';
 import { ConfigContext } from '../../../infrastructure/services/contextProviders/ConfigProvider';
+import getDateTimeFormat from '../../../application/valueObjects/dateTime/getDateTimeFormat';
 
 type useConfigProviderProps = {
-	dateFormat: boolean;
-	email: boolean;
-	timezone: boolean;
+	dateTimeFormat: boolean;
 };
 
 /**
  *
  * useConfigProvider is used to cherry-pick the exact configs we might need from ConfigProvider.
  */
+const useConfigProvider = ({ dateTimeFormat }: useConfigProviderProps) => {
+	const { dateFormat, timeFormat } = useContext(ConfigContext);
```

Comment:
Instead of `useContext(ConfigContext)`, we could create a hook like `useConfiguration()` or `useConfig()` to return that context.